consensus/polybft/signer/common.go

```diff
@@ -0,0 +1,139 @@
+package bls
```

Comment:
None of these common methods are covered with tests

Reply:
Although not directly covered, it is indirectly covered, since these functions are some helper functions for BLS keys management.

Suggesting leaving it as it is currently, since we are anyways going to consolidate bls keys management into a single implementation.